src/main/java/org/stratumproject/fabric/tna/behaviour/upf/FabricUpfProgrammable.java

```diff
@@ -959,4 +965,14 @@ private void applyUplinkRecirculation(Ip4Prefix subnet, boolean remove) {
             flowRuleService.removeFlowRules(denyRule, allowRule);
         }
     }
+
+    private boolean gettableFlowEntry(FlowEntry flowEntry) {
+        return flowEntry.state().equals(FlowEntry.FlowEntryState.PENDING_ADD) ||
+                flowEntry.state().equals(FlowEntry.FlowEntryState.ADDED);
+    }
+
+    private boolean gettableMeter(Meter meter) {
```

Comment:
Java method overloading can be helpful to reduce verbosity
```suggestion
    private boolean isHereToStay(Meter meter) {
```
```suggestion
    private boolean inserted(Meter meter) {
```